### PR TITLE
review project

### PR DESCRIPTION
**Pull request by JayKowski**
project complete, ready for review

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under the review by Ahmed Mahfoudh TSE

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,1281 @@
...
+}
+
+.footer-logo::after {
+  content: " ";
+  display: block;
+  clear: both;
+}
+
+.footer-links li {
+  /* width: 80%; */
```
Comment (stratospherique) [inline css/main.css]: https://github.com/JayKowski/new-york-times/blob/ffcb4f462975829b071e2214a98b2c7f49f3fd18/css/main.css#L828-L830

- Delete commented code

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Splendid work guys :clap: :clap: . However, some changes that prevents your project's approval needs to be made:

##### Footer

* [ ] It should have a fixed position (it sticks to the bottom part of the screen).

- You could consider a fixed footer like the image below but with less height
image

#### Implementation details

* [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/). For the `aside` section you are using absolute positioning :no_entry_sign: . Use floats :eyeglasses:

#### General notes:

* [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.


### That's it :relieved: . When finsihed, kindly resubmit another code review request :pray: . Keep up the good work :muscle: :muscle: .

**Review by uche-inyama — APPROVED** [review]
### Hello team! nice work. Job well-done. :raised_hands: :rocket: 
#### Go ahead and merge to master branch :smile: 
- [x] Your project doesn't have linter checks, please, ensure your setting are implemented the correct way.
### Keep up the good work :+1:

**Pull request merged**